tests/Metadata/Property/Factory/AnnotationPropertyNameCollectionFactoryTest.php

```diff
@@ -49,7 +49,7 @@ public function testCreate(PropertyNameCollectionFactoryInterface $decorated = n
         $this->assertEquals($results, iterator_to_array($metadata));
     }
 
-    public function getDependencies()
+    public function dependenciesProvider()
```

Comment:
this change is to avoid conflicting with a [parent class method added in PHPUnit 7.2](https://github.com/sebastianbergmann/phpunit/blob/7.2/src/Framework/TestCase.php#L962)
_method signature weren't compatible_